### ISSUE TITLE
Reddit app is broken due to domain being blocked

### ISSUE THREAD

**Issue by mineminemine** [anudeepND/blacklist#178]
<div align="center">  
  <img width="280" height="70" src="https://raw.githubusercontent.com/anudeepND/blacklist/master/images/blacklist-logo.png" alt="logo" />
</div>

<h2 align="center">Issue Submit Form</h2>
<div align="center">
  Provide the following info properly, which will help me to resolve your issue quickly.
</div>

### Issue(s):
Type `x` in between `[ ]` and make sure there isn't any space between brackets. Example; for Your Selected Issue(s), type like this - `[x]` 
You can select more than one category of issues if you need to!

- [ ] Whitelist
- [x] Blacklist
- [x] App Broken
- [ ] Website Broken
- [ ] Request
- [ ] Other Issue

<hr>

### Domain(s):
If you are submitting this issue for whitelist/blacklist issue, send me the domain(s) for whitelisting/blacklisting here. Kindly use the **Code Tag** to prevent tracking.

- Domain(s):

```
e.reddit.com
```

<hr>

### Details:
You can attach any screen shot or log of the issue or advert, this will help to highlight it.

- Your Issue Detail: 
The official Reddit app seems to heavily rely on e,reddit.com to work. If the domain is blocked, the app will not load any feed nor will it let the user log into the app. I have not done any further testing but just the two functions mentioned previously breaks the app altogether.

**Comment by anudeepND** [comment]
@mineminemine That's interesting, I reinstalled reddit app on my phone and I was able to sign in again and my feed was also working fine. I could see `e.reddit.com` getting blocked in pi-hole logs while doing these actions.

**Comment by mineminemine** [comment]
Hello @anudeepND it's been a while since I came across this issue but the last time I did, `e.reddit.com` was indeed the issue. However, I just tried it again on my app with `e.reddit.com` un-whitelisted and it seems like its working fine now, whatever was the problem seems to have been rectified on Reddit's side. Maybe they removed whatever feature that was relying on that specific subdomain.

So I guess this issue is now fixed, I'll just close the issue then. Sorry for the trouble.
